lib/community/widgets/community_sidebar.dart

```diff
@@ -117,7 +117,11 @@ class _CommunitySidebarState extends State<CommunitySidebar> {
                       child: ListView(
                         shrinkWrap: true,
                         children: [
-                          CommonMarkdownBody(body: communityView.community.description ?? ''),
+                          CommonMarkdownBody(
+                            body: communityView.community.description ?? '',
+                            // The sidebar is 0.8 the width of the screen, so we'll be just slightly smaller than that
+                            imageMaxWidth: 0.7 * MediaQuery.of(context).size.width,
```

Comment:
I think a slightly better approach here would be to extract the sidebar width constant so that it can be used here. The container width is determined by `FractionallySizedBox`'s `widthFactor` parameter on line `77`. We can create a constant for that value and use it here.

This will reduce overhead in having to adjust two different values (`widthFactor` and `imageMaxWidth`) if we ever want to change the width of the sidebar! Thoughts?

```dart
const kSidebarWidthFactor = 0.8;

...

FractionallySizedBox(widthFactor: kSidebarWidthFactor)

...
CommonMarkdownBody(
    body: communityView.community.description ?? '',
    imageMaxWidth: (kSidebarWidthFactor * MediaQuery.of(context).size.width) - somePadding,
),

```

Reply:
Good idea, done!